src/components/wasteCalendar/WasteCalendarLegend.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { StyleSheet, View } from 'react-native';
+import { normalize } from '../../config';
+
+import { WasteTypeData } from '../../types';
+import { RegularText } from '../Text';
+import { WrapperHorizontal, WrapperRow, WrapperWrap } from '../Wrapper';
+
+type EntryProps = {
+  color: string;
+  label: string;
+  selectedColor: string;
+};
+
+const Dot = ({ color }: { color: string }) => {
+  return <View style={[styles.dot, { backgroundColor: color }]} />;
+};
+
+const WasteCalendarLegendEntry = ({ color, label, selectedColor }: EntryProps) => {
+  return (
+    <WrapperHorizontal>
+      <WrapperRow>
+        <RegularText small>{`${label} - `}</RegularText>
```

Comment:
i would say we do not need that `-` here. the name followed by dots looks good here

Reply:
i don't really have a preference here, so i removed the dash